elliott/elliottlib/cli/find_bugs_golang_cli.py

```
@@ -182,12 +182,17 @@ def _is_fixed(self, bug: JIRABug, tracker_fixed_in: Set[Version], go_nvr_map) ->
-                    and existing_version.patch >= fixed_version.patch
+                # We only want to compare versions with the same major and minor version
+                # This is because backported patches are specific, e.g. 1.22.Y-Z can have a fix which is not present in 1.23.A-B
+                if (existing_version.major, existing_version.minor) != (fixed_version.major, fixed_version.minor):
+                    continue
+
+                if existing_version.patch >= fixed_version.patch and int(existing_version.prerelease) >= int(
+                    fixed_version.prerelease
                 ):
-                    self._logger.info(f"{bug.id} for {bug.whiteboard_component} is fixed in {str(existing_version)}")
+                    self._logger.info(
+                        f"{bug.id} for {bug.whiteboard_component} is considered fixed in {str(existing_version)} since it's greater than or equal to the fixed version {str(fixed_version)} mentioned in the tracker bug."
```

Ruff (0.15.6)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)